daemon/emer-main.c

```diff
@@ -219,13 +219,18 @@ on_authorize_method_check (GDBusInterfaceSkeleton *interface,
 
   const gchar *sender_name = g_dbus_method_invocation_get_sender (invocation);
   PolkitSubject *subject = polkit_system_bus_name_new (sender_name);
+  PolkitCheckAuthorizationFlags flags = POLKIT_CHECK_AUTHORIZATION_FLAGS_NONE;
+  GDBusMessage *message = g_dbus_method_invocation_get_message (invocation);
+
+  if (g_dbus_message_get_flags (message) & G_DBUS_MESSAGE_FLAGS_ALLOW_INTERACTIVE_AUTHORIZATION)
```

Comment:
Does gnome-control-center set this flag?

(Unfortunately I don't think Bustle supports showing this flag so my usual techniques to find out for myself don't work.)